src/wapm.spec.ts

```diff
@@ -55,42 +56,87 @@ test.serial('registry url', async ctx =>
   });
 });
 
-test('will login', async ctx =>
+if(userToken) {
+
+  test('will login with token', async ctx =>
 {
-  //Logout
-  logout();
+  // Logout
+  // logout();
 
   //Assert that it will not throw an error
   await ctx.notThrowsAsync(async () =>
   {
     const client = getClient(registry);
     //Login
-    await login(client, username, password);
+    await tokenLogin(client, userToken);
   });
 
   //Assert that we're logged in
   ctx.true(await loggedIn());
-
+  
   //Logout
-  logout();
+  // logout();
 });
 
-test('will publish', async ctx =>
-{
-  //Login
-  const client = getClient(registry);
-  await login(client, username, password);
-
-  //Get the fixture
-  const fixture = resolveFixture();
-
-  //Assert that it will not throw an error
-  await ctx.notThrowsAsync(async () =>
+  test('will publish with token', async ctx =>
   {
-    //Publish (Dry run)
-    await publish(fixture, true);
+    //Login
+    const client = getClient(registry);
+    await tokenLogin(client, userToken);
+  
+    //Get the fixture
+    const fixture = resolveFixture();
+  
+    //Assert that it will not throw an error
+    await ctx.notThrowsAsync(async () =>
+    {
+      //Publish (Dry run)
+      await publish(fixture, true);
+    });
+  
+    //Logout
+    // logout();
```

Comment:
I know I didn't originally do this, but I would call `logout` before and after every test to prevent tests from interfering with each other (You could just call `logout` in the [`test.beforeEach`](https://github.com/wasmerio/wapm-publish/blob/9a642e9d48b040505d250f6bfb1cce67bf77e59e/src/wapm.spec.ts#L23) and [`test.afterEach`](https://github.com/wasmerio/wapm-publish/blob/9a642e9d48b040505d250f6bfb1cce67bf77e59e/src/wapm.spec.ts#L27) hooks).